cmake/FindUnwind.cmake

```diff
@@ -6,8 +6,8 @@
 
 include (FindPackageHandleStandardArgs)
 
-find_path (Unwind_INCLUDE_DIR NAMES unwind.h libunwind.h DOC "unwind include directory")
-find_library (Unwind_LIBRARY NAMES unwind DOC "unwind library")
+find_path (Unwind_INCLUDE_DIR NAMES libunwind.h DOC "libunwind include directory")
```

Comment:
This is a breaking change that cannot be accepted.

Reply:
Here we search for `libunwind.h` and `libunwind.so` and will define `HAVE_LIB_UNWIND` if found and use `unw_*` API in code. `unwind.h` does not declare `unw_*` but `_Unwind_*`.

Reply:
This may be true. However, `FindUnwind.cmake` is intended to locate `unwind.h` not `libunwind.h`. The former is included by `stacktrace_x86_64-inl.h`:
https://github.com/google/glog/blob/9733727be4aed835e21a8f73c403f546b71663fe/src/stacktrace_x86_64-inl.h#L35

This is why I argue this change is not acceptable.

Reply:
> FindUnwind.cmake is intended to locate unwind.h not libunwind.h

It is not the current logic.

https://github.com/google/glog/blob/9733727be4aed835e21a8f73c403f546b71663fe/CMakeLists.txt#L88-L93


And  `HAVE_LIB_UNWIND` takes precedence over `HAVE_UNWIND_H`. So `unwind.h` is not used if `Unwind_FOUND`.

https://github.com/google/glog/blob/9733727be4aed835e21a8f73c403f546b71663fe/src/utilities.h#L108-L114

Reply:
Okay, but how are these changes (and renaming of headers) are related to Android support?

Reply:
only `unwind.h` is shipped with Android NDK.

Reply:
I renamed `stacktrace_x86_64-inl.h` to `stacktrace_unwind-inl.h` as it actually uses `_Unwind_*` API rather than hand-coded like `stacktrace_x86-inl.h`.

Reply:
Changes to `FindUnwind.cmake` actually are not related to Android support. Just correct it.

Reply:
Got it. I prefer moving `FindUnwind.cmake` changes to a separate PR.

Reply:
ok. what about the renaming change?

Reply:
I would revert that too for now. But I agree that we should probably rename the header to make its purpose clearer. Maybe in the same PR as changes to `FindUnwind.cmake`.